src/dex/balancer-v2/balancer-v2.ts

```diff
@@ -18,26 +18,25 @@ import {
   MAX_INT,
   MAX_UINT,
   Network,
+  subgraphTimeout,
```

Comment:
I think this was renamed in `constants.ts`, but not here and other files.

Reply:
Fixed